concent_api/core/tests/test_unit_model.py

```diff
@@ -374,9 +374,9 @@ def test_that_incorrect_version_of_golem_messages_in_stored_message_should_raise
 
             message_timestamp = parse_timestamp_to_utc_datetime(get_current_utc_timestamp())
             report_computed_task_message = StoredMessage(
-                type=force_report_computed_task.report_computed_task.header.type_,
+                type=force_report_computed_task.report_computed_task.header.type_,  # pylint: disable=no-member
                 timestamp=message_timestamp,
-                data=force_report_computed_task.report_computed_task.serialize(),
+                data=force_report_computed_task.report_computed_task.serialize(),  # pylint: disable=no-member
```

Comment:
This is completely not related with this commit.

Reply:
ok...